scripts/install

```diff
@@ -96,7 +96,7 @@ ynh_add_nginx_config
 ynh_script_progression --message="Configuring PHP-FPM..." --weight=2
 
 # Create a dedicated PHP-FPM config
-ynh_add_fpm_config
+ynh_add_fpm_config --phpversion=$phpversion
```

Comment:
`--phpversion=$phpversion` is not neeeded if you use `YNH_PHP_VERSION` in _common.sh

https://github.com/YunoHost/yunohost/blob/6fc6a2ba4c17434803cea2553a3dc7c51cedd338/data/helpers.d/php#L83-L84

Reply:
nope

Reply:
I fixed it I believe.